projects/ngx-formentry/src/form-entry/question-models/question-base.ts

```diff
@@ -42,6 +42,7 @@ export class QuestionBase implements BaseOptions {
   componentConfigs: Array<any>;
   options?: any;
   questionIndex?: Number;
+  message?: string;
```

Comment:
I don't love having a message as a property of a question. How do we translate this?

Reply:
@ibacher removed the alert feature to be implemented in a follow up PR